### PR TITLE
chore: add java example code for secrets manager

### PR DESCRIPTION
**Pull request by pratik151192**
Updated secrets manager page with Java example. I have also added a new helper on the sdk package with similar logic and tested that it worked succesfully. Currently waiting on a [PR](https://github.com/momentohq/client-sdk-java/pull/291) on the sdk package to be approved and merged before I cherry-pick the secrets manager commit on my local.

### CODE REVIEW

File: docs/develop/integrations/aws-secrets-manager.md [hunk 1]
```diff
@@ -110,6 +110,65 @@ async function run() {
 run();
 ```
 
+  </TabItem>
+  <TabItem value="java" label="Java" default>
+
+```java
```
Comment (cprice404) [inline docs/develop/integrations/aws-secrets-manager.md:116]: hey this is really cool! thanks for doing it. I would like to show you something about our docs tomorrow morning, I think there might be a better place for us to put this code. I'll ping you and we can find a time to zoom

Comment (pratik151192) [reply docs/develop/integrations/aws-secrets-manager.md:116]: Cool! have also added a new helper on the sdk package examples with similar logic and tested that it worked succesfully. Currently waiting on 
 the previous [PR](https://github.com/momentohq/client-sdk-java/pull/291) on the sdk package to be approved and merged before I cherry-pick the secrets manager commit on my local.

Comment (NoSQLKnowHow) [reply docs/develop/integrations/aws-secrets-manager.md:116]: @cprice404 so we are holding off on this PR until you get this code into the SDK repo and be able to link back to this?

Comment (cprice404) [reply docs/develop/integrations/aws-secrets-manager.md:116]: that is what I want to discuss with Pratik, yes.

Comment (pratik151192) [reply docs/develop/integrations/aws-secrets-manager.md:116]: yes am closing this and will generate a new PR with pages just linking to the examples under SDK folders.

**Pull request closed without merging**